permit_bb_put/__init__.py

```diff
@@ -0,0 +1,74 @@
+""" permit_bb init file """
+import os
+import json
+import logging
+import traceback
+import requests
+import jsend
+import azure.functions as func
+from requests.models import Response
+from shared_code.common import func_json_response, validate_access
+from shared_code.oracle import get_oracle_connection
+from shared_code.pts import get_pts_out
+
+#pylint: disable=too-many-locals
+def main(req: func.HttpRequest) -> func.HttpResponse:
+    """
+    main function for permit_post
+    """
+    logging.info('Permit Bluebeam PUT processed a request.')
+
+    try:
+        validate_access(req)
+        response = Response()
+        json_root = "message"
+        out = None
+        headers = {
+            "Access-Control-Allow-Origin": "*"
+        }
+
+        if req.get_body() and req.get_json() and len(req.get_json()):
+            response.status_code = 200
+            req_json = req.get_json()
+
+            sp_param = [
+                "P_APPLICATION_NUMBER",
+                "P_BLUEBEAM_PROJ_NO",
+                "P_STATUS",
+                "P_MSG"
+            ]
```

Comment:
The order of this array matters, right?  Should be defined according to the parameters of the stored proc?

Reply:
Right. This is for the Bluebeam update procedure. I believe that's the right parameter order?